tests/integration-tests/configs/common/common.yaml

```diff
@@ -81,6 +81,48 @@ cloudwatch_logging:
         instances: {{ common.INSTANCES_DEFAULT_ARM }}
         oss: {{ common.OSS_COMMERCIAL_ARM }}
         schedulers: ["slurm"]
+  test_compute_console_output_logging.py::test_console_output_with_monitoring_disabled:
+    dimensions:
+      - regions: ["us-east-1"]
+        instances: {{ common.INSTANCES_DEFAULT_X86 }}
+        oss: {{ common.OSS_COMMERCIAL_X86 }}
```

Comment:
Do we need to run all OSes. We are just checking the IAM roles, right?

Reply:
No we do not. Changed to "alinux2".